docs/examples/use_cases/tensorflow/efficientdet/docker/docker_run.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python
+
+import argparse
+import os
```

Comment:
Similarly to `SAMPLE_COMMAND` file and `docker_build.py` file, I'm having hard time to grasp, why is it here in the first place? If the only purpose would be to encapsulate `docker run` command, maybe we can include this command in the `README`?

`prepare_coco_dataset_docker.py` also falls into this category.